### ISSUE TITLE
Support for INET fields

### ISSUE THREAD

**Issue by gabssnake** [oguimbal/pg-mem#49]
This might not be so common, but we really like using [`INET`](https://www.postgresql.org/docs/9.1/datatype-net-types.html) type.
Any plans to support it or should we consider a PR ?
Best,

```sql
CREATE TABLE example (
  public_ip_address               INET,
  default_address_pool            INET NOT NULL DEFAULT '172.25.0.0/16'::INET,
);
```

**Comment by gabssnake** [comment]
Oh, good eye on the `,`, didn't notice it right away. This is great news :)

**Comment by oguimbal** [comment]
Should be fixed in `pg-mem@1.6.1` 😊
